index.js

```diff
@@ -3,7 +3,16 @@ const escapeStringRegexp = require('escape-string-regexp');
 
 const reCache = new Map();
 
-function makeRe(pattern, shouldNegate) {
+global.options = {
+	caseSensitive: false
+};
+
+function makeRe(pattern, shouldNegate, options) {
+	if (options !== undefined) {
```

Comment:
```js
const opts = Object.assign({}, options, {
    caseSensitive: false
});
```

Reply:
@SamVerschueren  The options and defaults above are in the wrong order above, but @Ricardo-Silva91 put it the correct way. Please review and approve.